src/lib/create-config.ts

```diff
@@ -182,17 +183,63 @@ const dateHandlingCallback = (connection, callback) => {
     });
 };
 
+const databaseSslFromFile = (caFilePath: string) => {
+    try {
+        return readFileSync(caFilePath).toJSON();
+    } catch (e) {
+        return {};
```

Comment:
I'd be tempted to just bubble here. If you asked for this to happen and it didn't, I'd be rather surprised to find a quiet fail without a log

Reply:
Agreed. Let's fail hard, we do in other paths for config.